components/automate-ui/src/app/pages/api-token/list/api-token-list.component.html

```diff
@@ -29,13 +29,19 @@
     <div class="page-body">
       <chef-toolbar>
         <app-authorized [anyOf]="[['/iam/v2beta/tokens', 'post'], ['/auth/tokens', 'post']]">
-          <chef-button id="create-button" primary (click)="openCreateModal()">Create Token</chef-button>
+          <div *ngIf="(sortedApiTokens$ | async)?.length === 0" class="token-empty-state">
+            <p>Create the first token to get started!</p>
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Token</chef-button>
+          </div>
+          <div *ngIf="(sortedApiTokens$ | async)?.length > 0">
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Token</chef-button>
+          </div>
         </app-authorized>
       </chef-toolbar>
       <app-authorized [anyOf]="[['/iam/v2beta/tokens', 'get'], ['/auth/tokens', 'get']]">
         <chef-table>
           <chef-thead>
-            <chef-tr>
+            <chef-tr *ngIf="(sortedApiTokens$ | async)?.length > 0">
```

Comment:
💭 I wonder why we'd put this on `chef-tr` and not one of the parent tags? `app-authorized` or `chef-table` or `chef-thead`